src/app.js

```diff
@@ -1 +1,36 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function copyFileToDestination() {
+  const [src, dest] = process.argv.slice(2);
+
+  if (process.argv.length < 4) {
+    console.log('Please enter file name and destination to copy');
+
+    return;
+  }
+
+  if (src === dest) {
+    console.log('You are trying to copy file to the same destination');
+
+    return;
+  }
+
+  const source = path.join(__dirname, src);
+  const destination = path.join(__dirname, dest);
```

Comment:
if i want to copy file not from current directory the code will brake, you dont need to use dirname